CHANGELOG.md

```diff
@@ -0,0 +1,174 @@
+# CHANGELOG
+
+This is a manually generated log to track changes to the repository for each release. 
+Each section should include general headers such as **Implemented enhancements** 
+and **Merged pull requests**. All closed issued and bug fixes should be 
+represented by the pull requests that fixed them. This log originated with Singularity 2.4
+and changes prior to that are (unfortunately) done retrospectively. From @olifre, critical items to know are:
+
+ - renamed commands
+ - deprecated / removed commands
+ - changed defaults
+ - backward incompatible changes (recipe file format? image file format?)
+ - migration guidance (how to convert images?)
+ - changed behaviour (recipe sections work differently)
+
+
+## [v3.0](https://github.com/singularityware/singularity/tree/development) (development)
+
+**Implemented enhancements**
+ - This changelog was added.
+
+**Bug Fixes**
+ - .wh.* files leading to permissions errors from Docker aufs extractions are skipped on extraction
```

Comment:
Let's keep changes out until they are actually merged.  Otherwise we get into a situation where the changelog represents hopes instead of reality.